SETUP/CHANGELOG.md

```diff
@@ -7,10 +7,52 @@ see the git history.
 [R202009](https://github.com/DistributedProofreaders/dproofreaders/releases/tag/R202009)
 first before upgrading to R202102 or later releases.**
 
-## R......
-
+## R202403
+Scripts supporting this upgrade are in `SETUP/upgrade/20`
+
+### Notices & Deprecations
+This is the last release to explicitly support MySQL 5.7. Future releases may
+continue to work with this version but only 8.x will be tested in the future.
+
+This is the last release to include most `faq/*` content. These documents have
+not been maintained for quite some time as pgdp.net has moved them to their wiki.
+
+The following globals are deprecated and will be removed in the next release:
+* `$Activity_for_id_`
+* `$Pool_for_id_`
+* `$Stage_for_id_`
+* `$Round_for_round_id_`
+* `$Round_for_round_number_`
+* `$Round_for_project_state_`
+* `$Round_for_page_state_`
+* `$PAGE_STATES_IN_ORDER`
+
+The following constant is deprecated and will be removed in the next release:
+* `MAX_NUM_PAGE_EDITING_ROUNDS`
+
+### Changes
 * Session management by cookie has been removed in lieu of PHP-based sessions
   which have been the DP code default since before 2004 (cpeel)
+* New My Suggestions page (cpeel)
+* New `Cyrillic` character suite (srjfoo)
+* Highlight non-Latin Unicode scripts in WordCheck (cpeel)
+* SAs allowed to delete illustrations at any stage (srjfoo)
+* Improvements for file uploads and validation (70ray, cpeel)
+* Smoothreading notification improvements (70ray)
+* Format Preview updates (70ray)
+* Several new APIs (bpfoley) -- see [`dp-openapi.yaml`](../api/dp-openapi.yaml)
+  for complete spec
+  * `v1/queues`
+  * `v1/queues/:queueid`
+  * `v1/queues/:queueid/stats`
+  * `v1/queues/:queueid/projects`
+  * `v1/stats/site/projects/stages`
+  * `v1/stats/site/projects/states`
```

Comment:
Might be too small to mention, but the following too
```
* `v1/projects` -- add `in_smoothreading` flag and `project_manager` field
*  `v1/projects/:projectid/transitions`
*  `v1/projects/:projectid/pagedetails add `username` field

Reply:
I'll add `v1/projects/:projectid/transitions` as a new endpoint but leave the new fields out for this doc.